ports/esp32/panichandler.c

```diff
@@ -0,0 +1,42 @@
+#include "genhdr/mpversion.h"
```

Comment:
Can you please add a license/copyright header?  You can copy it from, say, `esp32_partition.c` but replace the copyright line with your name and the current year.